pkg/apis/operator/v1alpha1/glustercluster_types.go

```diff
@@ -1,22 +1,60 @@
 package v1alpha1
 
 import (
+	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
 )
 
-// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
-// NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
+// GlusterStorageTarget defines a storage target
+type GlusterStorageTarget struct {
+	Name      string       `json:"name,omitempty"`
+	Addresses []string     `json:"address"`
+	Creds     *Credentials `json:"credentials,omitempty"`
+}
+
+// GlusterClusterReplicationDetails defines replication details
```

Comment:
Please explain what "replication details" are used for.

Reply:
Ack